apps/condo/domains/organization/schema/OrganizationNewsItem.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Generated by `createschema organization.OrganizationNewsItem 'title:Text; body:Text; type:Select:common,emergency; receiversCriteria?:Json; meta?:Json;'`
+ */
+
+const { Relationship, Select, Text } = require('@keystonejs/fields')
+
+const { Json } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/organization/access/OrganizationNewsItem')
+const { NEWS_TYPES } = require('@condo/domains/organization/constants')
+
+
+const OrganizationNewsItem = new GQLListSchema('OrganizationNewsItem', {
```

Comment:
probably we need a new `news` domain

Reply:
Or a miniapp, as discussed a bit earlier. 🤔

Reply:
Decided to move into the new domain.